CONTRIBUTING.md

```diff
@@ -9,10 +9,16 @@ The following is a set of guidelines for contributing to the SkillUP Quiz projec
 [Code of Conduct](#code-of-conduct)
 
 [I don't want to read this whole thing, I just have a question!!!](#i-dont-want-to-read-this-whole-thing-i-just-have-a-question)
+
 [What should I know before I get started?](#what-should-i-know-before-i-get-started)
 
+<<<<<<< HEAD
 * [Git & Github](#git-and-github)
```

Comment:
Hey @abhishekakade, let it be only `[GitHub](#github)`. If you are sure that it is working by this way then it is fine otherwise I would suggest you to change it to only `Github`.